### PR TITLE
tests(increase-coverage): weekly tests PR 

### PR DESCRIPTION
**Pull request by IbrahimCSAE**
## Context

This is a weekly tests PR

## Changes

- Converted all spec.js to spec.ts
- Added simulateClicksOnEliminate utility, with an option to delay between clicks, useful for clicks that generate rendering operations like segmentation where a delay is needed

## Tests

- should render and allow usage of circle brush
- should render and allow usage of circle eraser
- should render and allow usage of sphere brush
- should render and allow usage of sphere eraser
- should render and allow usage of threshold circle
- should render and allow usage of rectangle scissor
- should render and allow usage of circle scissor
- should render and allow usage of sephere scissor
- should render and allow usage of eraser scissor
- should render and allow usage of paint fill
- surface segmentation representation for volume viewports

## Results

image

### CODE REVIEW

**Comment by sedghi** [comment]
WIP or can i review

File: packages/dicomImageLoader/src/imageLoader/registerLoaders.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+console.debug("dicom loader: link: init: 61D6BD83-8191-47A6-8049-67277CED2B48");
```
Comment (sedghi) [inline packages/dicomImageLoader/src/imageLoader/registerLoaders.ts:1]: revert

Comment (IbrahimCSAE) [reply packages/dicomImageLoader/src/imageLoader/registerLoaders.ts:1]: haha sorry lol

**Review by sedghi — APPROVED** [review]
Thanks, looks great

**Comment by IbrahimCSAE** [comment]
CI is failing so lets not merge until I fix this, not sure whats the cause yet since it passes locally

**Review by sedghi — APPROVED** [review]
Thanks, great stuff

**Pull request merged**